File: Python/baekjoon/단계별로 풀어보기/조건문/14681_사분면 고르기.py
# https://www.acmicpc.net/problem/14681

x = False if input().startswith('-') else True
y = False if input().startswith('-') else True
if x and y:
    print(1)
elif not x and y:
    print(2)
elif not x and not y:
    print(3)
else:
    print(4)

File: C#/Baekjoon/조건문/14681_사분면 고르기.cs
//https://www.acmicpc.net/problem/14681
int x = int.Parse(Console.ReadLine());
int y = int.Parse(Console.ReadLine());
if (x > 0)
{
    Console.WriteLine((y > 0) ? "1" : "4");
}
else
{
    Console.WriteLine((y > 0) ? "2" : "3");
}